include/exercise_functions.php

```diff
@@ -0,0 +1,472 @@
+<?php
+
+function addExercise(){
+
+  $exerciseTypeId = getExerciseTypeId();
```

Comment:
Also a note about error handling. Even when we expect things to work, its good to build in clauses to catch (and handle gracefully) something not working, so instead of your app crashing maybe it can throw a nice error message. So like here, what if for some reason we weren't able to find an `exerciseTypeId` we don't want to just continue on with the function (esp because it'll make a db query with maybe bad data) so we'd want to throw a check here to make sure we actually got a value. 

You don't necessarily want to have to check every value is actually set, so you will definitely want to be discerning about where you add that to.